.devcontainer/devcontainer.json

```
@@ -1,28 +1,31 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options,
 // see documentation located at the following URL:
 // https://github.com/OpenINF/docker-fisher/tree/main/library-scripts/docs
-// For format details, see https://aka.ms/devcontainer.json. For config options, see the
-// README at: https://github.com/devcontainers/templates/tree/main/src/docker-outside-of-docker
+// For config options, see the README at:
+// https://github.com/devcontainers/templates/tree/main/src/docker-outside-of-docker
```

```suggestion
// For config options, see the README at:
// https://github.com/devcontainers/templates/tree/main/src/docker-outside-of-docker
```

my mistake, why link there &mdash; not the first place we should send folks